include/z64actor.h

```diff
@@ -5,6 +5,7 @@
 #include "z64math.h"
 #include "z64animation.h"
 #include "z64collision_check.h"
+#include "z64scene.h"
```

Comment:
needed?

Reply:
I don't know. Actually I'm not sure why these are in here as I don't remember doing them myself.

Reply:
Hmm, if you didn't add it specifically I would say whatever is in main should be fine and this should be removed.

Reply:
ok, took these things out. everything seems to work fine without them.